packages/legend-application-studio/style/index.scss

```diff
@@ -17,6 +17,7 @@
 @forward 'components/not-found-screen';
 @forward 'components/workspace-setup';
 @forward 'components/editor';
+@forward 'components/editor/database-editor';
```

Comment:
this shouldnt be needed. 
You need to add the forward in the file `style\components\_editor.scss` use the runtime-editor.scss as reference how to add a scss style